src/graph-helpers/relationship.js

```diff
@@ -0,0 +1,39 @@
+import relationship from './raw-relationship';
+import { parseArgs } from './raw-relationship';
+import fields from './fields';
+import join from './join';
+
+function wrapCallback(callbackFunction) {
+  return function (schema) {
+    if (schema.name.match(/Connection$/)) {
+      const pageInfo = relationship(schema, 'pageInfo', pageInfoSchema => {
+        return fields(pageInfoSchema);
+      });
+      const edge = relationship(schema, 'edges', edgeSchema => {
+        const edgeFields = fields(edgeSchema);
+        const node = relationship(edgeSchema, 'node', nodeSchema => {
+          return callbackFunction(nodeSchema);
+        });
+
+        return join(edgeFields, node);
```

Comment:
because at that point, I'm relying on the implementer to use the same join everywhere. I would need everyone working on this to guarantee that the join characters are `', '`, which is not something I want to have to do, or build a linting rule around. This normalizes the pattern so nobody needs to care about what characters to use when joining.